assets/ui/components/ArticleHeadline.jsx

```diff
@@ -3,7 +3,9 @@ import PropTypes from 'prop-types';
 
 export default function ArticleHeadline({item}) {
     return (
-        item.headline && <h2 className='wire-column__preview__headline'>{item.headline}</h2> || null
+        item.es_highlight && item.es_highlight.headline ? <div
+            dangerouslySetInnerHTML={({__html: item.es_highlight.headline[0]})}
+        /> : item.headline && <h2 className='wire-column__preview__headline'>{item.headline}</h2> || null
```

Comment:
imo we should always end up with:
```
<h2 className='wire-column__preview__headline'>
```
so if there are highlights it should go inside that h2 so we keep the styling, same for slugline